tests/integration/standard/test_metadata.py

```diff
@@ -1103,12 +1103,13 @@ def test_legacy_tables(self):
 
 CREATE TABLE legacy.composite_comp_no_col (
     key blob,
-    column1 'org.apache.cassandra.db.marshal.DynamicCompositeType(org.apache.cassandra.db.marshal.TimeUUIDType, org.apache.cassandra.db.marshal.BytesType, org.apache.cassandra.db.marshal.UTF8Type)',
```

Comment:
I am aware of the analog between "static" composites and clustering. However, I don't think DynamicCompositeType has a native CQL equivalent. I thought the only way to get the DCT behavior was to use the internal class/alias literal.